publish-scripts/chocolatey/installps_template

```diff
@@ -40,11 +40,12 @@ $packageArgs = @{
 Install-ChocolateyZipPackage @packageArgs
 
 # only symlink for func.exe
-$files = Get-ChildItem $toolsDir -include *.exe -recurse
+$files = Get-ChildItem $toolsDir -filter *.exe -Recurse -File
```

Comment:
Update the PR description to reflect the latest changes (We are still using recursive approach, with additional checks)